### PR TITLE
Modifying Rolling the Average

### PR DESCRIPTION
**Pull request by 1sourcecontrol**
#312 
I tried to rewrite the text to make it explicit that crits are ignored, and clarified the text for percentile rolls.

### CODE REVIEW

**Comment by 1sourcecontrol** [comment]
This ability may be a little strong - ignoring critical fails - but the ability is explicitly stated to be uncommonly used,  so I figure it will be fine.

File: 01_01_Basic_Rules.txt [hunk 1]
```diff
@@ -57,11 +57,10 @@ and you just roll once.
 
 Rolling the Average:
 	There are certain circumstances and feats which allow for Rolling the Average.
-This must be announced before the die are cast. Rolling the average will give you 
-the advantage of avoiding a critical fail, but also prevents you from critically 
-succeeding. This is best used in situations when massive failure is not an option. To do
-this, you roll 2d10s, add them together, and divide by 2. The resulting number is your
-roll. 
+This must be announced before the die are cast. Critical fails or Critical successes
+are ignored when Rolling the Average. This is best used in situations when massive
```
Comment (bleehu) [inline 01_01_Basic_Rules.txt]: Non-blocking: I'd like to leave strategic advice out of the basic rules. It's the player's job to come up with their tactical analysis. Even if that means making their own mistakes.

**Review by bleehu — APPROVED** [review]
Notes, but no issues. 
Shipit!

File: 01_01_Basic_Rules.txt (hunk 1 above)
Comment (1sourcecontrol) [reply 01_01_Basic_Rules.txt]: This was from @Reveraine's first PR (that you approved).

I'll make a mental note for the future

Comment (1sourcecontrol) [reply 01_01_Basic_Rules.txt]: > It's the player's job to come up with their tactical analysis

Although, in retrospect, I think that's personal opinion and not fact.  I've seen some games give strategy tips that I greatly appreciated. And sometimes players really struggle with games, so leaving a section (not necessarily in-line with rules definitions) for strategy tips could be a boon.

**Pull request merged**